### ISSUE TITLE
invalid shortlink target 

### ISSUE THREAD

**Issue by andreh7** [pytorch/tutorials#100]
The link https://goo.gl/uGOksc referring to a document 'Train neural nets to play video games' redirects to a non-existing notebook https://github.com/pytorch/tutorials/blob/master/Reinforcement%20(Q-)Learning%20with%20PyTorch.ipynb 

This link appears in these files: 

- https://github.com/pytorch/tutorials/blob/01b89c9387203dddbfbaeb9dce047df4db62d22c/beginner_source/former_torchies/parallelism_tutorial.py#L120
- https://github.com/pytorch/tutorials/blob/01b89c9387203dddbfbaeb9dce047df4db62d22c/beginner_source/blitz/cifar10_tutorial.py#L312 . 

Probably what it should link to is https://github.com/pytorch/tutorials/blob/master/intermediate_source/reinforcement_q_learning.py . 

(if this is correct and you want to eliminate the short url link like for the other links in these files, I can provide a pull request if wanted)

**Comment by jnhwkim** [comment]
In the same page, `Chat with other users on Slack` is also broken in [the web page](http://pytorch.org/tutorials/beginner/former_torchies/parallelism_tutorial.html).

**Comment by chsasank** [comment]
@andreh7, It'd indeed be great if you can raise a PR. Sorry for late reply.

**Comment by andreh7** [comment]
As you see in this thread, for the issue reported by @jnhwkim , I submitted pull request #102.

For the issue originally reported by me, I propose pull request #104, I'm new to sphinx so I hope this is in the spirit of the rest of the documentation. 

I also noticed that the title of the link mentions video games while the the actual target is about the cart pole problem but the original link was also about reinforcement q learning.

Let me know if I should change something in the pull requests.
